### PR TITLE
feat!: add support for olmo release

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
## Description 
This PR adds support for the latest Open edX + eduNEXT release.

### Questions
Should we disable the account MFE as well? Which other MFEs are now turned on by default?
- Account
- Learning
- Auth
should we disable all from the tutor-mfe list?
See the answer in: https://github.com/eduNEXT/tutor-contrib-edunext-distro/pull/42#issuecomment-1461278273

### CODE REVIEW

File: tutordistro/plugin.py [hunk 1]
```diff
@@ -121,7 +121,7 @@
             "index": "git",
             "name": "eox-audit-model",
             "repo": "eox-audit-model",
-            "version": "v1.0.0",
+            "version": "v2.0.0",
             "domain": "github.com",
             "protocol": "https",
             "path": "eduNEXT",
```
Comment (MaferMazu) [inline tutordistro/plugin.py:127]: We need to change nuez tag by olmo tag in this file (ex: the theme and the docker overrides)

**Review by MaferMazu — COMMENTED** [review]
The rest looks good to me.

About your question:
As far as I understand distro plugin doesn't disable the use of MFE by default; but if someone wants to disable MFE the solution is run `tutor plugins disable mfe` and add in the configs `DISTRO_DISABLE_MFE: true`. In favor of Tutor Olive enables tutor-mfe by default (have installed or not), maybe it's a good idea to add in the [README](https://github.com/eduNEXT/tutor-contrib-edunext-distro#how-disable-mfe) that if you don't want mfe, you need to run the disable command too.

**Comment by dcoa** [comment]
For olive only learning MFE is mandatory.

 Tutor-mfe plugin has a command to [disable individual MFE](( https://github.com/overhangio/tutor-mfe#disabling-individual-mfes. ) but not learning, that's why we create this feature, to disable learning MFE as well. Now that learning is mandatory and is something that we want for Olmo release I think we can remove this feature.

File: tutordistro/plugin.py (hunk 1 above)
Comment (mariajgrimaldi) [reply tutordistro/plugin.py:127]: on it! thanks

**Comment by MaferMazu** [comment]
If we remove the MFE feature, can we remove the info in the [readme](https://github.com/eduNEXT/tutor-contrib-edunext-distro#how-disable-mfe)?
Please, @mariajgrimaldi, let me know when you change this.

**Review by MaferMazu — DISMISSED** [review]

**Review by JuanDavidBuitrago — DISMISSED** [review]

**Pull request merged**